tests/e2e/testfiles/scenarios.yaml

```diff
@@ -478,7 +482,6 @@
     - "authorization"
     - "powerscale"
     - "observability"
-    # - "sanity"
```

Comment:
I understand removing a commented-out tag, but should this tag actually be existent? What are the guidelines for a sanity test, and should this be included among them?

Reply:
That is a great question and the quick answer is, not to my knowledge. This is something that can be improved in future releases.